src/test/e2e/05_tarball_test.go

```diff
@@ -29,13 +30,28 @@ func TestMultiPartPackage(t *testing.T) {
 	e2e.CleanFiles(deployPath, outputFile)
 
 	// Create the package with a max size of 1MB
```

Comment:
nit: comment wasn't updated to reflect what the code is doing

I'd also be okay with deleting the comment because it's self-explanatory IMO

Other than that, this lgtm

```suggestion
	// Create the package with a max size of 20MB
```